From 5e0ca52c644ee63111efa5c0db514efbb0af3e52 Mon Sep 17 00:00:00 2001
From: Peter van Dijk <peter@7bits.nl>
Date: Fri, 10 Mar 2023 23:58:52 +0000
Subject: use right /dev/ path on Haiku

merged upstream as https://github.com/tio/tio/pull/189 so this
patch can be removed on the next tio release

diff --git a/src/tty.c b/src/tty.c
index f5ffc35..406ab1d 100644
--- a/src/tty.c
+++ b/src/tty.c
@@ -62,6 +62,9 @@
 #elif defined(__CYGWIN__)
 #define PATH_SERIAL_DEVICES "/dev/"
 #define PREFIX_TTY_DEVICES "ttyS"
+#elif defined(__HAIKU__)
+#define PATH_SERIAL_DEVICES "/dev/ports/"
+#define PREFIX_TTY_DEVICES ""
 #else
 #define PATH_SERIAL_DEVICES "/dev/serial/by-id/"
 #define PREFIX_TTY_DEVICES ""
-- 
2.37.3

